package.json

```diff
@@ -83,7 +85,7 @@
     "@types/jest": "^26.0.20",
     "@types/json-stable-stringify": "^1.0.32",
     "@types/log-symbols": "^3.0.0",
-    "@types/node": "^12.18.0",
+    "@types/node": "^14.16.0",
```

Comment:
Good catch!

Reply:
I found it because otherwise I got warnings trying to log `os.version()`. Apparently that's new in 14 :slightly_smiling_face: